docker/entrypoint.sh

```diff
@@ -21,6 +21,7 @@ else
   exit 1
 fi
 
-$PIP install -f /pip "tensorflow==$TENSORFLOW_VERSION" "keras==$KERAS_VERSION" h5py model-converters
-$PYTHON /usr/local/bin/keras_to_tensorflow "/model_input/$MODEL_INPUT_FILENAME" "/model_output/$MODEL_OUTPUT_DIRNAME"
-chmod -R 0777 /model_output
+$PIP install -f /pip "tensorflow==$TENSORFLOW_VERSION" "keras==$KERAS_VERSION" h5py
+mkdir /input && stored sync "$MODEL_INPUT" /input/model.hdf5
```

Comment:
`stored` _should_ `mkdir` already here

Reply:
It does for the `local` backend (https://github.com/triagemd/stored/blob/master/stored/backends/local.py#L32-L34) but not for either `gs` (https://github.com/triagemd/stored/blob/master/stored/backends/gs.py#L49) or `http` (https://github.com/triagemd/stored/blob/master/stored/backends/http.py#L13)